onnxruntime/server/executor.cc

```diff
@@ -27,30 +28,30 @@ namespace protobufutil = google::protobuf::util;
 protobufutil::Status Executor::SetMLValue(const onnx::TensorProto& input_tensor,
                                           MemBufferArray& buffers,
                                           OrtAllocatorInfo* cpu_allocator_info,
-                                          /* out */ MLValue& ml_value) {
+                                          /* out */ Ort::Value& ml_value) {
   auto logger = env_->GetLogger(request_id_);
 
   size_t cpu_tensor_length = 0;
-  auto status = onnxruntime::utils::GetSizeInBytesFromTensorProto<0>(input_tensor, &cpu_tensor_length);
+  auto status = onnxruntime::server::GetSizeInBytesFromTensorProto<0>(input_tensor, &cpu_tensor_length);
```

Comment:
There is `OrtGetTensorMemSizeInBytesFromTensorProto`.

Reply:
I chose not to use those for two reasons:
1. Those work on serialized protobufs and for us to support JSON and GRPC we need to be able to control the deserialization.
2. I got the impression from Changming that we should treat the TensorProto as internal to ORT and not depend specifically on it. @snnn ?